### ISSUE TITLE
disableLogs not working

### ISSUE THREAD

**Issue by Nailik** [tanersener/flutter-ffmpeg#115]
Hello,
when i call `FlutterFFmpegConfig().disableLogs()` i suggest i will see no more logs.

But when i use `FlutterFFprobe().getMediaInformation()` i still get a very long output

(Running on Android 10, buildSdk 28)

Example:
```
I/mobile-ffmpeg(18287): Input #0, mp3, from '/storage/emulated/0/Music/Imagine Dragons - Smoke + Mirrors (2015)/14. Thief.mp3':
I/mobile-ffmpeg(18287):   Metadata:
I/mobile-ffmpeg(18287):     album           : 
I/mobile-ffmpeg(18287): Smoke + Mirrors (Limited Super Deluxe Edition)
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     artist          : 
I/mobile-ffmpeg(18287): Imagine Dragons
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     disc            : 
I/mobile-ffmpeg(18287): 2/2
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     genre           : 
I/mobile-ffmpeg(18287): Rock
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     publisher       : 
I/mobile-ffmpeg(18287): Interscope (Universal Music)
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     title           : 
I/mobile-ffmpeg(18287): Thief
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     track           : 
I/mobile-ffmpeg(18287): 14/21
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     date            : 
I/mobile-ffmpeg(18287): 2015
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):   Duration: 
I/mobile-ffmpeg(18287): 00:03:47.74
I/mobile-ffmpeg(18287): , start: 
I/mobile-ffmpeg(18287): 0.025056
I/mobile-ffmpeg(18287): , bitrate: 
I/mobile-ffmpeg(18287): 321 kb/s
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     Stream #0:0
I/mobile-ffmpeg(18287): : Audio: mp3, 44100 Hz, stereo, fltp, 320 kb/s
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     Metadata:
I/mobile-ffmpeg(18287):       encoder         : 
I/mobile-ffmpeg(18287): LAME3.99r
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     Side data:
I/mobile-ffmpeg(18287):       
I/mobile-ffmpeg(18287): replaygain: 
I/mobile-ffmpeg(18287): track gain - 
I/mobile-ffmpeg(18287): -9.600000
I/mobile-ffmpeg(18287): , 
I/mobile-ffmpeg(18287): track peak - 
I/mobile-ffmpeg(18287): unknown
I/mobile-ffmpeg(18287): , 
I/mobile-ffmpeg(18287): album gain - 
I/mobile-ffmpeg(18287): unknown
I/mobile-ffmpeg(18287): , 
I/mobile-ffmpeg(18287): album peak - 
I/mobile-ffmpeg(18287): unknown
I/mobile-ffmpeg(18287): , 
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     Stream #0:1
I/mobile-ffmpeg(18287): : Video: mjpeg, yuvj420p(pc, bt470bg/unknown/unknown), 500x500 [SAR 1:1 DAR 1:1]
I/mobile-ffmpeg(18287): , 
I/mobile-ffmpeg(18287): 90k tbr, 
I/mobile-ffmpeg(18287): 90k tbn, 
I/mobile-ffmpeg(18287): 90k tbc
I/mobile-ffmpeg(18287):  (attached pic)
I/mobile-ffmpeg(18287): 
I/mobile-ffmpeg(18287):     Metadata:
I/mobile-ffmpeg(18287):       comment         : 
I/mobile-ffmpeg(18287): Cover (front)
I/mobile-ffmpeg(18287): 

```

How can i disable this?

**Comment by tanersener** [comment]
Logs printed are not coming from `flutter_ffmpeg`, they're from the parent library `mobile-ffmpeg`. But yeah, I think you're right. You shouldn't be seeing them after calling `disableLogs()`.

You can use the following call to disable them until I publish an update.

```
setLogLevel(LogLevel.AV_LOG_WARNING);
```

**Comment by Nailik** [comment]
Thank you very much, this works as workaround.

**Comment by tanersener** [comment]
Fixed in development

**Comment by tanersener** [comment]
Fixed in `v0.3.0`.
